api/PayrollProcessor.Data.Persistence/Features/Departments/DepartmentEmployeeRecord.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using PayrollProcessor.Core.Domain.Features.Departments;
+using PayrollProcessor.Core.Domain.Features.Employees;
+using PayrollProcessor.Data.Persistence.Infrastructure.Records;
+
+namespace PayrollProcessor.Data.Persistence.Features.Departments
+{
+    public class DepartmentEmployeeRecord : CosmosDBRecord
+    {
+        public Guid EmployeeId { get; set; }
+        public string Department { get; set; } = "";
+        public string FirstName { get; set; } = "";
+        public string LastName { get; set; } = "";
+        public string Email { get; set; } = "";
+
+        public static class Map
```

Comment:
I'm finding these nested `Map` classes to be an interesting alternative to `AutoMapper`.

Our domain/persistence layers are pretty similar right now, so AutoMapper would likely handle the mapping fine without much ceremony.

There's something nice about the explicit `Map` methods/functions compared to some of the magic of `AutoMapper`.

Thoughts?

Reply:
I think this works great as an alternative to automapper, expecially since we know exactly which types we will be mapping to and from. No dynamic mapping for us.